client/blueprints/task/files/lib/__name__/app/controllers/overlays/__name__.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import TaskController from 'tahi/pods/task/controller';
+
+export default TaskController.extend();
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.